### ISSUE TITLE
Windows

### ISSUE THREAD

**Issue by spot92** [ProGamerGov/Neural-Tile#5]
If anyone has a Windows version of this, that would be awesome. Otherwise, I will start on one.

**Comment by audreyls** [comment]
I have an [installation guide to make this sort of thing work in Windows here](https://github.com/audreyls/neuralstyle-tf-masterscript/wiki/Windows-Specific-Installation-Guide). It's still through shell scripts, but there's lots of options to get them to work in Windows. The repository the wiki is in has scripts that I wrote to work with that guide and should work out of the box. If you're looking for something truly specific to Windows or something though, yeah, don't know.

**Comment by spot92** [comment]
I'll look into this. If this works, you're an MVP, otherwise you're just a VP

**Comment by ProGamerGov** [comment]
There appears to be a way to install bash on Windows via Windows 10's linux subsystem, but I've heard others doing something like converting the scripts to batch file (`.bat`).

**Comment by spot92** [comment]
@audreyls does your windows method tile?

**Comment by audreyls** [comment]
Yes, you can see the script's functionality described in the repository's
readme. It's fairly extensive. The installation guide is just setting up
getting the backbone of neural style to work in Windows, such that the
repository's scripts can be used. Feel free to comment with issues over there if you'd like.

**Comment by ProGamerGov** [comment]
@spot92 Are you running Windows 10? If so, then there's actually a feature which will add all the Linux stuff like bash/shell scripts to your Windows 10 install: 

https://docs.microsoft.com/en-us/windows/wsl/install-win10

https://itsfoss.com/install-bash-on-windows/

https://www.windowscentral.com/how-install-bash-shell-command-line-windows-10
